### DISCUSSION TITLE
Magicbell in aapanel

### DISCUSSION THREAD

**Discussion by ehssanehs** [magicbell/community#180] [Questions] [answered]
Hi everybody
I have my website using aapanel on linux machine.
I want to use magicbell inbox on my index.php page.
So may I know how to code for showing it in my page step by step?
Sorry because I am new to web developing.
I installed nodejs and npm on my linux but as I found it is needed to install them by aapanel and I did it but I dont know how to use it in my project.

**Comment by unamashana** [comment, marked as answer]
Unfortunately, I am not sure I understand your question, but it sounds like you need some more general programming experience before using MagicBell. If you aren't familiar, I'd start with the book "Eloquent Javascript".
